### ISSUE TITLE
Bookmarks disappear when VS Code is reopened

### ISSUE THREAD

**Issue by PeterMacej** [alefragnani/vscode-bookmarks#562]
Type: <b>Bug</b>

I searched all similar issues and I think mine is the same as #538.

**To reproduce:**
1. Create a simple file c:\test.txt with a single line "abc". It's a standalone file, not a part of a project, folder, or workspace.
2. Toggle a bookmark on this line. The bookmark is correctly created and works fine.
3. Close VS Code.
4. Open the file in VS Code. The bookmark is missing.

**Expected behavior:**
The bookmark should be visible in step 4.

I don't use Pretier.
I did some tests and inspected the VS Code storage with DB Browser for SQLite. After I closed VS Code, the bookmark data was still present in <AppData>\Roaming\Code\User\workspaceStorage\1666599132300\state.vscdb:

![bookmark data](https://user-images.githubusercontent.com/10990713/197482911-0df97d3f-5922-483a-96aa-a46e32c27034.png)

When I re-opened the file in VS Code and then closed VS Code, the data was removed:

![no bookmark data](https://user-images.githubusercontent.com/10990713/197483166-55769915-2ae5-4448-8db8-a8c809264ff8.png)

So the problem occurs when VS Code or the file is open.

Extension version: 13.3.1
VS Code version: Code 1.72.2 (d045a5eda657f4d7b676dedbfa7aab8207f8a075, 2022-10-12T22:15:18.074Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: No

**Comment by mjoe67886** [comment]
Yeah I'm getting the same issue I thought I was deleting accidentally with the clear function but I just reopened and my bookmarks are gone
